### PR TITLE
Drive cx.ex drive cx

### PR DESCRIPTION
**Pull request by michal-keboola**
changing the license file and adding README with information for obtaining API secret key

### CODE REVIEW

File: resources/kds-team.ex-procore/templates/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Data Services CZ
```
Comment (JakubMatejka) [inline resources/kds-team.ex-procore/templates/LICENSE]: Just a triviality, you would probably want to unify the license files with the one above.

Comment (JakubMatejka) [reply resources/kds-team.ex-procore/templates/LICENSE]: But  I wonder, why do you want to include the licenses here anyway?

Comment (michal-keboola) [reply resources/kds-team.ex-procore/templates/LICENSE]: Wait, what? :) I lived under the impression the MIT license has to be there. In that case, I'll just remove it then.

Thank you for looking into this!!!

Comment (JakubMatejka) [reply resources/kds-team.ex-procore/templates/LICENSE]: This repository is just a technical infrastructure for delivering the templates. We can move it to some other place or even integrate it into the Developer Portal itself. It should not be a display window so I think it's not necessary. ;o)

**Pull request closed without merging**